tests/tests.go

```diff
@@ -227,7 +235,18 @@ func (c *TestContext) getWholeGraph(t *testing.T, at time.Time) string {
 func (c *TestContext) getAllFlows(t *testing.T, at time.Time) string {
 	gremlin := g.G.Context(at).V().Flows().Sort()
 
-	flows, err := c.gh.GetFlows(gremlin)
+	var flows []*flow.Flow
```

Comment:
If we are here, it means the fail has already failed. In this case, we want to dump the flows that we had at that time (as much as possible). With the `Retry`, we could dump some flows that were not present when the test failed, we would definitely not help to understand the issue

Reply:
the retry is to avoid "request in the future"  note the retry is for 1sec max

Reply:
after another pass, I definitively forgot/cleanup to remove this part